release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -75,6 +75,16 @@ Beginning with {product-title} {product-version}, you can specify either Network
 
 For more information, see xref:../installing/installing_aws/installing-aws-network-customizations.adoc[Installing a cluster on AWS with network customizations].
 
+[id="ocp-4-12-aws-local-zones-customization"]
+==== Extend worker nodes to the edge of AWS when installing into an existing VPC with Local Zone subnets.
+
+Now you can install {product-title} to an existing VPC with installer-provisioned infrastructure, extending the worker nodes to Local Zones subnets. The installer will provision worker nodes on the edge of the AWS network, with NoSchedule taints to be dedicated to the user applications.
+
+Applications deployed on the Local Zones locations deliver low latency for end users. When deploying with the ALB ingress controller, applications running on {product-title] worker nodes can serve requests with custom ingress.
```

Comment:
@GroceryBoyJr is it a typo? `{product-title]`  -> `{product-title}` 

@lihongan can you help check this section? thanks

Reply:
The "ALB ingress controller" and last statement sound ambiguous, does it mean ALB operator or just ingress controller ?  The ingress controller on AWS supports CLB and NLB but not ALB. To support ALB we need to install ALB operator (see https://github.com/openshift/aws-load-balancer-controller and https://github.com/openshift/aws-load-balancer-operator/)

Indeed applications running on edge worker nodes can serve requests with the "default" ingresscontroller and `low latency for end users` is true.

Reply:
@yunjiang29 and @lihongan thank you so much for your review! Editing now!

Reply:
@lihongan and @yunjiang29 I can delete the entire second sentence, since the details seem too complicated for a release note!